hyrisecockpit/database_manager/background_scheduler.py

```diff
@@ -415,22 +413,32 @@ def _update_storage_data(self) -> None:
                 "storage", {"storage_meta_information": dumps(output)}, time_stamp
             )
 
+    def _get_origin_tables_from_workload_type(
+        self, table_names: List[str], workload_type: str
+    ):
+        return [table_name.split(f"_{workload_type}")[0] for table_name in table_names]
+
     def _generate_table_loading_queries(
-        self, table_names: List[str], folder_name: str
+        self, table_names: List[str], workload_type: str
```

Comment:
Could this be renamed back to `folder_name`? We adhered to this naming everywhere.
```suggestion
        self, table_names: List[str], folder_name: str
```